### PR TITLE
fix: remove bigint type from response of getTransaction api 

### PR DESCRIPTION
**Pull request by Minhaj-ul-abidin**
as  bigint is not bson serialisable

### CODE REVIEW

**Comment by thakursachin467** [comment]
@Minhaj-ul-abidin can you format the response of `getTransaction ` similar to how we do it with celo https://github.com/Transak/celo-transak/blob/main/index.js#L103

File: index.js [hunk 1]
```diff
@@ -100,10 +100,11 @@ async function getTransaction(hash, network) {
         }
         return response
     } catch (e) {
+        console.log(e)
         // throw(e.message)
     }
 }
-
+getTransaction("d6a4e99a0c23f6931765121592ef7746107ee49920374ffe836ee04efeba7ec3","main").then(console.log)
```
Comment (agarwalyeshu) [inline index.js]: We should receive this right ?

**Review by agarwalyeshu — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (Minhaj-ul-abidin) [reply index.js]: Right fixing this.

**Comment by Minhaj-ul-abidin** [comment]
Changes already pushed in #6

**Pull request closed without merging**